modules/virt-aws-bm.adoc

```diff
@@ -31,7 +31,12 @@ Before you set up your cluster, review the following summary of supported featur
 
 Installing::
 --
+ifndef::openshift-rosa[]
 * You can install the cluster by using installer-provisioned infrastructure, ensuring that you specify bare-metal instance types for the worker nodes by editing the `install-config.yaml` file. For example, you can use the `c5n.metal` type value for a machine based on x86_64 architecture.
+endif::openshift-rosa[]
+ifdef::openshift-rosa[]
+* You can install the cluster by using installer-provisioned infrastructure, ensuring that you specify bare-metal instance types for the worker nodes. For example, you can use the `c5n.metal` type value for a machine based on x86_64 architecture.
+endif::openshift-rosa[]
```

Comment:
Does it make sense to minimize the amount of repeated content (in case edits are needed in that part later?). 

Something like:
```suggestion
* You can install the cluster by using installer-provisioned infrastructure, ensuring that you specify bare-metal instance types for the worker nodes. For example, you can use the `c5n.metal` type value for a machine based on x86_64 architecture.
ifndef::openshift-rosa[]
You specify bare-metal instance types by editing the `install-config.yaml` file
endif::openshift-rosa[]
```